src/main/java/com/conveyal/r5/analyst/scenario/CustomModification.java

```diff
@@ -0,0 +1,83 @@
+package com.conveyal.r5.analyst.scenario;
+
+import com.conveyal.r5.transit.TransportNetwork;
+import com.fasterxml.jackson.annotation.JsonAnyGetter;
+import com.fasterxml.jackson.annotation.JsonAnySetter;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * This is the R5 (worker) model object for a custom freeform JSON modification. Rather than JSON properties of a
+ * specific type of modification being mapped to Java fields, this class puts them in a freeform Map from String keys
+ * to Objects. Unlike all other R5 Modification subclasses, this modification type is not intended to ever be applied
+ * to a TransportNetwork. It is used only in the Analysis backend. When it is serialized by the backend and sent to a
+ * worker, the type property will be changed from "custom" to whatever is in the r5type property.
```

Comment:
Reading this through helps clarify the question I had in the PR comment thread. Should we rename this class `CustomModificationTemplate` or `CustomModificationHolder` to make the intended use clearer?

Reply:
Good point. It really is an exception in many ways to the usual Modification classes. I'm thinking if we move this to analysis-backend it will make it even more abundantly clear that this Modification just can't even exist in an R5 worker. I'll rename it, then we can potentially move it to the backend after discussion.

Reply:
Oh, actually it would be kind of messy to move to analysis-backend because the class needs to appear in ModificationTypeResolver. I'll just rename it for clarity.